tango_simlib/examples/mkat_vds.py

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env python
 
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
+from __future__ import absolute_import, division, print_function
+from future import standard_library
+standard_library.install_aliases()
```

Comment:
@amakhaba This requires a `#noqa` for the python3 see https://github.com/ska-sa/katcp-python/blob/4240fb45ec8cb73c4b60490e6e43ffe97700f567/doc/conf.py#L17